Improve Migration Guide for YoutubeAudioSourceManager (#18)
lavalink-devs/youtube-source: README.md

    client enabled, age-restricted tracks are **not** guaranteed to play.

## Migration from Lavaplayer's built-in YouTube source

This client is intended as a direct replacement for Lavaplayer's `YoutubeAudioSourceManager`,

> which may be deprecated in more recent releases of Lavaplayer.

Reviewer: ```suggestion
which has been deprecated in a recent releases of Lavaplayer.
```

> deprecated in release [2.1.2](https://github.com/lavalink-devs/lavaplayer/releases/tag/2.1.2).

Reviewer 2: I wouldn't include a hard reference to this specific deprecation as other forks may not necessarily have deprecated the source, or they might've deprecated it separately as is the case with my fork.

By default, Lavaplayer instantiates and registers an instance of its `YoutubeAudioSourceManager` unless
explicitly excluded.

First, create and register an instance of the supported `YoutubeAudioSourceManager` from the `youtube-source` package.
```java
AudioPlayerManager playerManager = new DefaultAudioPlayerManager();

> AudioSourceManager ytSourceManager = new dev.lavalink.youtube.YoutubeAudioSourceManager();

Reviewer: ```suggestion
YoutubeAudioSourceManager ytSourceManager = new dev.lavalink.youtube.YoutubeAudioSourceManager();
```

                                          com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.class);
```

> Then, create and register an instance of the supported `YoutubeAudioSourceManager` from the `youtube-source` package.

Reviewer 2: Move this section above the first codeblock. The new source manager should be registered before the others, as `registerRemoteSources` includes the `HttpAudioSourceManager` which catches all identifiers passed to it, meaning that the youtube source would never get queried when used like this.
